public/docs/ts/latest/guide/router.jade

```diff
@@ -1236,30 +1423,77 @@ a#can-activate-guard
 
   The `CanActivate` guard is the tool to manage these navigation business rules.
 
-  #### Add a crisis admin feature
+  #### Add an admin feature module
 
   We intend to extend the Crisis Center with some new *administrative* features.
-  Those features aren't defined yet. So we add the following placeholder component.
+  Those features aren't defined yet. So we add a new feature module named `AdminModule`.
+  We'll follow our same convention by creating an `admin` folder with a feature
+  module file, route file and supporting components.
+
+  Our admin feature module file structure looks like this:
+
+.filetree
+  .file app/admin
+  .children
+    .file admin-dashboard.component.ts
+    .file admin.component.ts
+    .file admin.module.ts
+    .file admin.routing.ts
+    .file manage-crises.component.ts
+    .file manage-heroes.component.ts
+
+:marked
+  Our admin feature module contains our `AdminComponent` used for routing within our
+  feature module, a dashboard route and two unfinished components to manage crises and heroes.
+
++makeTabs(
+   `router/ts/app/admin/admin-dashboard.component.1.ts,
+   router/ts/app/admin/admin.component.ts,
+   router/ts/app/admin/admin.module.ts,
+   router/ts/app/admin/manage-crises.component.ts,
+   router/ts/app/admin/manage-heroes.component.ts
+  `,
+  null,
+   `app/admin/admin-dashboard.component.ts,
+   app/admin/admin.component.ts,
+   app/admin/admin.module.ts,
+   app/admin/manage-crises.component.ts,
+   app/admin/manage-heroes.component.ts
+  `)
 
-+makeExcerpt('app/crisis-center/crisis-admin.component.1.ts')
+.l-sub-section
+  :marked
+    Since our admin dashboard `RouterLink` is an empty path route in our `AdminModule`, it
+    is considered a match to any route within our admin feature area. We only want the `Dashboard`
+    link to be active when we visit that route. We've added an additional binding to our `Dashboard` routerLink,
+    `[routerLinkActiveOptions]="{ exact: true }"` which will only mark the `./` link as active when
+    we navigate the to `/admin` URL and not when we navigate to one the other child routes.
 
 :marked
-  Next, we add a child route to the `crisis-center.routes` with the path, `/admin`.
+  Our initial admin routing configuration:
 
-+makeExcerpt('app/crisis-center/crisis-center.routing.3.ts (admin route)', 'admin-route-no-guard')
++makeExcerpt('app/admin/admin.routing.1.ts (admin routing)', 'admin-routes')
+
+h3#component-less-route <i>Component-Less Route</i>: grouping routes without a component
+:marked
+  Looking at our child route under the `AdminComponent`, we have a route with a **path** and a **children**
```

Comment:
Yes, we could. I have a separate `AdminDashboardComponent` to have a default route for the admin routing. Otherwise we'd have to use a redirect to one of the other routes.